doc/website/release-notes/iceoryx-v3-0-0.md

```diff
@@ -0,0 +1,1499 @@
+# iceoryx v3.0.0
+
+## [v3.0.0](https://github.com/eclipse-iceoryx/iceoryx/tree/v3.0.0) (2025-02-02)
+
+[Full Changelog](https://github.com/eclipse-iceoryx/iceoryx/compare/v2.0.6...v3.0.0)
+
+**Notes**
+
+- The minimal supported GCC compiler is now 8.3
+- The required C++ standard is now C++17
+- Experimental 32-bit support for all platforms supporting 64-bit atomic operations
+- Experimental support for communication between 32-bit and 64-bit applications
+
+**Major changes:**
+
+- Address clang-tidy and [AUTOSAR C++14 ruleset](https://www.autosar.org/fileadmin/standards/R22-11/AP/AUTOSAR_RS_CPP14Guidelines.pdf) warnings
+- Extensive quality work restructuring, reworking and bugfixing of `iceoryx_hoofs`
+- Introduction of the new package `iceoryx_platform` for better platform abstraction and portability
+- Add Bazel as a build system
```

Comment:
@elBoberido

This is the short summary of the release. Please check if anything major is missing.

Reply:
There is some experimental API to graceful handle `Runtime` and `PublisherPort`/`SubscriberPort` errors more graceful and give the user a chance for recovery instead of terminating the application. But since it is not yet available for request-response, nor the C API, I'm hesitant to make this public to the big crowed.